src/browser/components/SettingsPage.jsx

```diff
@@ -426,6 +427,13 @@ const SettingsPage = createReactClass({
       <Row>
         <Col md={12}>
           <h2 style={settingsPage.sectionHeading}>{'App Options'}</h2>
+          <div className='IndicatorContainer'>
+            <AutoSaveIndicator
+              show={this.state.savingType === 'app'}
+              savingState={this.state.savingState}
```

Comment:
To take @jasonblais suggestion, you would need to manage different `state` parameters for each indicator.